package.json

```diff
@@ -42,6 +42,7 @@
   "devDependencies": {
     "babel-cli": "^6.7.5",
     "babel-eslint": "^7.1.1",
+    "babel-plugin-transform-object-rest-spread": "6.22.0",
```

Comment:
This devDependency shouldn't be needed - the object-rest-spread transform is included in the stage 3 preset, which is a transitive dependency of the stage 0 preset already in place. Does the build complete without this new dependency?